rootfs/etc/dconf/profile/user

```diff
@@ -0,0 +1,2 @@
+user-db:user
```

Comment:
what is this for?

Reply:
This is for setting default configurations systemwide for all users in the system. Like this:
https://help.gnome.org/admin/system-admin-guide/stable/dconf-custom-defaults.html.en
Note we are not locking them out, it can be changed. But sets the defaults in the local db.